src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -51,6 +51,19 @@ public ServiceManagerBuilder WithRouter(IEndpointRouter router)
             return this;
         }
 
+        // todo: make public
+        internal ServiceManagerBuilder WithNewtonsoftJsonHubProtocol(Action<NewtonsoftServiceHubProtocolOptions> configure)
+        {
+            _services.AddNewtonsoftHubProtocol(configure);
+            return this;
+        }
+
+        // todo: make public
+        internal ServiceManagerBuilder WithNewtonsoftJsonHubProtocol()
+        {
+            return WithNewtonsoftJsonHubProtocol(o => { });
```

Comment:
`AddNewtonsoftHubProtocol` to check `configure = null` instead?

Reply:
From [Framework Design Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/member-overloading)
> DO use member overloading rather than defining members with default arguments.

Reply:
I mean `WithNewtonsoftJsonHubProtocol(null);` as anyway user can pass `null` into the above method overload

Reply:
I think we could support `WithNewtonsoftJsonHubProtocol(null)`, but `WithNewtonsoftJsonHubProtocol()` seems more elegant, and just like dotnet core https://github.com/dotnet/aspnetcore/blob/main/src/SignalR/common/Protocols.NewtonsoftJson/src/NewtonsoftJsonProtocolDependencyInjectionExtensions.cs#L25

Reply:
I think this is a pattern about nullable, either allow null so you can pass in null directly or not allow null and you need to always pass in an object